README.md

```diff
@@ -56,11 +56,13 @@ To run the app locally, first clone this repository and `cd` into its directory.
     npm install
     ```
 
-1. Copy the `.env_example` file to `.env`, and edit it to include your [Twilio API credentials](https://www.twilio.com/console) and the phone number and TwimL App Sid you made above. We use [python-dotenv](https://github.com/theskumar/python-dotenv) to load those variables automatically.
```

Comment:
neither of the virtualenv steps above work for me

Reply:
Hey Bharat. Thanks for pointing this out. I'm going to update the Readme file with the template and add commands from the Makefile.